diff --git a/node_modules/react-native-calendars/src/expandableCalendar/WeekCalendar/new.js b/node_modules/react-native-calendars/src/expandableCalendar/WeekCalendar/new.js
index b7e3a65..839f8da 100644
--- a/node_modules/react-native-calendars/src/expandableCalendar/WeekCalendar/new.js
+++ b/node_modules/react-native-calendars/src/expandableCalendar/WeekCalendar/new.js
@@ -56,7 +56,7 @@ const WeekCalendar = (props) => {
         return (<Week {...others} key={item} current={isSameWeek ? date : item} firstDay={firstDay} style={weekStyle} markedDates={markedDates} onDayPress={onDayPress} context={context}/>);
     }, [date, markedDates]);
     return (<View testID={testID} style={[allowShadow && style.current.containerShadow, !hideDayNames && style.current.containerWrapper]}>
-      {!hideDayNames && (<View style={[style.current.week, style.current.weekCalendar]}>
+      {!hideDayNames && (<View style={[style.current.week, style.current.weekCalendar,{paddingLeft:0,paddingRight:0}]}>
           <WeekDaysNames firstDay={firstDay} style={style.current.dayHeader}/>
         </View>)}
       <View>
